samples/nrf9160/modem_shell/src/utils/mosh_print.c

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <assert.h>
+
+#include <zephyr.h>
+#include <posix/time.h>
+#include <shell/shell.h>
+#include <shell/shell_uart.h>
+
+#include "mosh_print.h"
+
+extern const struct shell *shell_global;
```

Comment:
I'm cleaning this up so that shell is obtained when the first mosh_fprintf call is made. This way the shell pointer is only required in mosh_print and in main(). The latter will be removed once a zephyr fix gets merged so that we don't need resize command in main().

Reply:
Sorry, actually we'll continue to need the shell resize command but can avoid the empty command after it.